### ISSUE TITLE
display quotes

### ISSUE THREAD

**Issue by r10s** [deltachat/deltachat-desktop#1921]
current master already has a new functionality for handling quotes, see https://github.com/deltachat/deltachat-core-rust/pull/1975 and android will probably have a quote-functionality in the next release.

_as a first step_, desktop should start **displaying quotes**. while in the past the sender was somehow aware of adding some context to the text, now one might do that eg. by quoting and just saying "yes". therefore, displaying quotes is pretty urgent to get in (in contrast to **creating quotes** which can also be a release or so later).

displaying a quote is pretty straight forward:

- use [`dc_msg_get_quoted_text()`](https://c.delta.chat/classdc__msg__t.html#a456299eae0b43c4ec6ee6fe812947ed9) and display the returned text in a quoting style as in the android version (slightly smaller text, vertical bar on the left, only display 2~3 lines of text.
- note, that quotes may be added to all types of messages
- if `dc_msg_get_quoted_text()` returns NULL, there is just no quote

this is already the absolute minimum. once that is done,

-  use [`dc_msg_get_quoted_msg()`](https://c.delta.chat/classdc__msg__t.html#a9282d97546388655d99018fd63294bb6) to enrich the quote:

- show quoted contact name above the quote using the contact-color
- also colorize the vertical bar in the contact-color
- a tap on the quote should jump upwards to the original message
- if `dc_msg_get_quoted_msg()` returns 0 while `dc_msg_get_quoted_text()` returns != NULL, the original message is unknown/deleted/not-yet-there, in this case just display the minimum from above

once this is done, we can continue enriching quotes by displaying eg. the quoted image. however, that can also be done in another ticket :)

**Comment by r10s** [comment]
to clarify: when using core45 or core46, this is not an issue as they show the quotes as part of the text returned by [`dc_msg_get_text()`](https://c.delta.chat/classdc__msg__t.html#a2577851182c3665a8b8a2de759fd09f1) - eg. current desktop 1.13.1 works fine with new android-version using quotes. 

but once a newer core is used, this should be targeted.
